### DISCUSSION TITLE
Nine patch extension for egui?

### DISCUSSION THREAD

**Discussion by bcolloran** [fishfolk/bones#283] [Q&A] [answered]
I have seen a couple reference to bones including a nine-patch extension for egui, but looking around your repo I haven't located that code. Does that exist? If so, where might I find that code?

Thanks!

**Comment by zicklag** [comment, marked as answer]
Hey there! We do indeed have 9-patch style buttons and frames for Egui:

https://github.com/fishfolk/bones/tree/main/framework_crates/bones_framework/src/render/ui/widgets
